assets/js/modules/ads/components/setup/SetupFormPAX.js

```diff
@@ -54,6 +55,17 @@ export default function SetupFormPAX( {
 	const { setConversionTrackingEnabled, saveConversionTrackingSettings } =
 		useDispatch( CORE_SITE );
 
+	const currentConversionID = useSelect( ( select ) =>
+		select( MODULES_ADS ).getConversionID()
+	);
+
+	const glaConnectedAdsID = useSelect( ( select ) =>
+		select( MODULES_ADS ).getGoogleForWooCommerceConversionID()
+	);
```

Comment:
What does "gla" stand for here? This variable isn't super-obvious to me and I don't see that abbreviation elsewhere in the codebase.